changelog.txt

```diff
@@ -1,3 +1,10 @@
+Version 5.1.1
+==============
+Bug fixes:
+- **When specifying a port, ADAL.NET would always make the call on port 443**. ADAL.NET now honors the port specified by the developer. [Issue 1627](https://github.com/AzureAD/azure-activedirectory-library-for-dotnet/issues/1627)
```

Comment:
This should be taken out. The feature has not been reviewed by anyone yet and is still in PR.
https://github.com/AzureAD/microsoft-authentication-library-for-dotnet/pull/1313